app/components/search-result.js

```diff
@@ -27,6 +27,14 @@ export default Ember.Component.extend(Analytics, {
         return result.description.slice();
     }),
 
+    contributors: Ember.computed('result', function() {
+        let result = this.get('result');
+        if (result.contributors) {
+            return result.contributors.reverse();
```

Comment:
I forgot we were doing this sorting preprints-side and thought it was coming from SHARE already in this order.